tests/Aspire.Hosting.MongoDB.Tests/MongoDbFunctionalTests.cs

```diff
@@ -259,6 +259,7 @@ public async Task VerifyWithInitBindMount()
             .Build();
 
         var bindMountPath = Path.Combine(Path.GetTempPath(), Path.GetRandomFileName());
+        Directory.CreateDirectory(bindMountPath);
```

Comment:
These 2 lines can be replaced with:

`var bindMountPath =  Directory.CreateTempSubdirectory().FullName;`